### PR TITLE
MOTECH 1968 - New lookups for CallDetailRecord, migrate type of motec…

### PR DESCRIPTION
**Pull request by MikolajBasSoldevelo**
…hTimestamp

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by pgesek** [comment]
test this please

File: ivr/src/main/java/org/motechproject/ivr/repository/CallDetailRecordDataService.java [hunk 1]
```diff
@@ -25,4 +25,18 @@
...
             @LookupField(name = "callStatus", customOperator = MATCHES_CASE_INSENSITIVE) String callStatus);
+
+    @Lookup
+    List<CallDetailRecord> findByProviderCallIdWithCaseSensitive(@LookupField(name = "providerCallId") String providerCallId);
+
+    @Lookup
+    List<CallDetailRecord> findByCallStatus(@LookupField(name = "callStatus") String callStatus);
+
+    @Lookup
+    List<CallDetailRecord> findByMotechTimestamp(@LookupField(name = "motechTimestamp", customOperator = "LIKE") String motechTimestamp);
```
Comment (pgesek) [inline ivr/src/main/java/org/motechproject/ivr/repository/CallDetailRecordDataService.java]: LIKE is an sql operator, 80% sure it doesn't work here . Did you test this?

Comment (MikolajBasSoldevelo) [inline ivr/src/main/java/org/motechproject/ivr/repository/CallDetailRecordDataService.java]: I see similar solution in one motech classes so i assume that it should work, its not tested.

Comment (MikolajBasSoldevelo) [inline ivr/src/main/java/org/motechproject/ivr/repository/CallDetailRecordDataService.java]: Thats right it's not working, is in mds operator which have similar fuctionality as LIKE?

**Comment by pgesek** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Pull request closed without merging**